canonical-web-and-design/dotrun
# Support settings in .env file

Specifically, allow the DOTRUN_DOCKER_COMPOSE_ACTIONS setting
to be set in a .env file.

This makes sense because whether you want the database to spin
up for specific actions will be a project-level decision,
not one for individual developers to set on their workstation.

For example, I want to set this to "start:serve:test:test-python"
in ubuntu-com-security-api so I can use the database for testing.

QA
--

```
snapcraft
snap install --dangerous dotrun_1.4.2_amd64.snap
```

Then `cd` to a project (e.g. ubuntu.com).
Edit `.env` to add:

```
DOTRUN_DOCKER_COMPOSE_ACTIONS=start:serve:test:test-python
```

The run `dotrun test-python`, and check the database starts.
